connect_ext_ppr/webapp.py

```diff
@@ -37,6 +39,37 @@
 )
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
 
+    @router.get(
+        '/deployments/{deployment_id}',
+        summary='Deployment details',
+        response_model=DeploymentSchema,
+    )
+    def get_deployment(
+        self,
+        deployment_id: str,
+        client: ConnectClient = Depends(get_installation_client),
+        db: VerboseBaseSession = Depends(get_db),
+        installation: dict = Depends(get_installation),
+    ):
+        dep = (
+            db.query(Deployment)
+            .filter_by(id=deployment_id, account_id=installation['owner']['id'])
+            .first()
+        )
+        if dep is None:
+            raise ObjectNotFound(deployment_id)
+
+        listings = get_all_info(client)
```

Comment:
May be this is not part of this PR but the name of the function 'get_all_info' seems not descriptive enough.

Reply:
Decided to fix it later